frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -94,6 +105,7 @@ function FileSearcher:onShowFileSearch(search_string)
     end
     UIManager:show(search_dialog)
     search_dialog:onShowKeyboard()
+    return true
```

Comment:
@Frenzie, do you want it removed from here too?

Reply:
Assuming of course it makes no difference on Kindle one way or the other. Otherwise it should stay.

Reply:
I mean, on kindle both `return true`s solve the problem. Does your update to base fix it on kindle too?

Reply:
No, in that case it should stay.

Reply:
Just to clarify, both stay then?

Reply:
Please (re)describe the problem you had - because it is probably not ours and we may not be thinking/talking about the same stuff :)
If it is that on Kindle you had that `s` inputed everytime you launch it (unlike us, only the first time), it's still something to undestand - and I'd like to know why these `return true` do help.

Reply:
Yes, on kindle, with the original proposal (Alphanumeric) any key press (to call search) would carry over to the search field, so you could literally start typing whatever you were searching for without waiting. So if you were searching for “Blimp” the first <kbd>B</kbd> press would both call up the  feature and put it in the field, so all you had to type left was “limp”.

with the tragic loss of that wonderful implementation (looking at someone who shall remain nameless) that behaviour is no longer desirable and the `return true` fixes it.

I can’t really tell you how “it does help”, what i can say is this. I was studying the other two functions (shortcuts introduced) that didn’t present that behaviour (dict and wiki) and the only major difference was that they were returning true to handler, so i tried it and it worked.

Reply:
OK. (So, if you remove the return tru efrom these two, they would also input D or W in their field?)

Reply:
correct, I only tested it with D but I imagine it would be the same with W